pkgs/by-name/mo/monophony/package.nix

```
@@ -0,0 +1,73 @@
+{ lib
+, fetchFromGitLab
+, python3Packages
+, wrapGAppsHook4
+, gst_all_1
+, gobject-introspection
+, gtk4
+, yt-dlp
+, libadwaita
+, libsoup_3
+, glib-networking
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "monophony";
+  version = "2.3.1";
+  format = "other";
+
+  sourceRoot = "source/source";
+  src = fetchFromGitLab {
+    owner = "zehkira";
+    repo = "monophony";
+    rev = "v${version}";
+    hash = "";
+  };
+
+  pythonPath = with python3Packages; [
+    mpris_server
+    pygobject3
+    ytmusicapi
+  ];
+
+  nativeBuildInputs =
+    [
+      gobject-introspection
+      wrapGAppsHook4
+    ]
+    ++ (with python3Packages; [ ordered-set nuitka ]);
```

```suggestion
    ++ (with python3Packages; [ nuitka ]);
```
@natsukium had me add `ordered-set` to nuitka's propagatedBuildInputs so it is not needed here.

---

Okay

---

Fixed